### ISSUE TITLE
Refreshing gtDelphiZXingQRCode in the process of generating multiple QRCodes

### ISSUE THREAD

**Issue by Elinulok32** [War3Evo/gtDelphi#5]
Hello, Thank you for your great work with gtDelphiZXingQRCode. It is an amazing work done and has has saved me a great deal of time.
I intend to use this to package to generate and save multiple QRCodes at the click of a button (say 20). I place the whole process in a thread and it works very well but at times the generation process freezes until I leave the TForm and come back to it. The freezing usually happens on Android  but doesn't on Windows. Is there a way to keep refreshing the "TgtQRCodeGenFMX" in the proccess, after it has generated about 5 QRcodes and then continue till it gets to the total quantity required without stopping? Your assistancec is urgently needed. Thank you.

**Comment by War3Evo** [comment]
## Background Task Setup in Android (Delphi)

Android can be challenging when it comes to allowing tasks to work in the background. To keep your task running, you'll need to set it up as a foreground service using intents.

### Important Consideration for Delphi Users
Delphi versions change frequently, and the Java intents in the code might need adjustments based on the specific version of Delphi you are using. It's recommended to be mindful of these changes and adapt the code accordingly.

### Example Code
I have an example code I used while working on the PLA Steamer app for Phone Losers of America prank call show. You can find the source code here:

[FMX.Radio Android Service Example](https://github.com/War3Evo/FMX.Radio/blob/master/AndroidServiceRadioSampleApplication/Unit1.pas)

### Setting Up as a Service
Once you set up the task as a foreground service, it should keep running without freezing or stopping. This is essential to ensure smooth background execution on Android.

**Comment by War3Evo** [comment]
Here's some instructions I wrote a while back on creating services that might interest you: https://github.com/War3Evo/FMX.Radio/tree/master#instructions

**Comment by Elinulok32** [comment]
Hello, Thank you very much for your rapid response. I realized that the problem was not with the generation of the QRCode using  gtDelphiZXingQRCode but rather the rendering of the generated SVG File using Skia4Delphi's TSkSvg. The TSkSvg renders it very well but because of the frequency and speed in the use of the same control, it freezes at some point. What I did to resolve this was to put: "SkSvg1.Svg.Source := sSVGfile;" in between "SkSvg1.BeginUpdate;" and "SkSvg1.EndUpdate;" and viola, no more freezing.

Thanks all the same for the hint on: "Background Task Setup in Android (Delphi)".
